lib/OriginalSource.js

```diff
@@ -41,10 +41,6 @@ class OriginalSource extends Source {
 				}
 				return new SourceNode(null, null, null,
 					_splitCode(line + (idx != lines.length - 1 ? "\n" : "")).map(function(item) {
-						if(/^\s*$/.test(item)) {
```

Comment:
This was from my previous attempt: #41. Removed, as it wasn't quite correct.

I think it worked in some cases, but only coincidentally. Possibly it made the first identifier on a line map correctly, and I didn't look further than that while checking it worked?